### DISCUSSION TITLE
如何获取含有@Bot的消息，但排除回复消息？

### DISCUSSION THREAD

**Discussion by SherkeyXD** [nonebot/discussions#539] [问答] [answered]
直接使用 `to_me()` 的 rule 会导致回复消息也被算进去
在想该怎么写

**Comment by yanyongyu** [comment, marked as answer]
如果你是ob适配器的话，你可以自己写一个rule，判断 `event.original_message`

**Reply by SherkeyXD** [reply]
解决了，感谢

~~一开始用 `event.get_message()` 发现at和回复没差别~~
